app/views/vet/confirmation.njk

```diff
@@ -16,8 +16,9 @@
       <p class="govuk-body">Invoice the farmer if you have not already done this.</p>
       <h2 class="govuk-heading-m">Contact the Rural Payments Agency</h2>
       <p class="govuk-body">If you have a question about your submission, call the Rural Payments Agency on 03000 200 301</p>
-      <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>.</p>
-      <p class="govuk-body"><a href="https://www.gov.uk/call-charges" target="_blank">What did you think of this service?</a>(takes 30 seconds)</p>
+      <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays</p>
```

Comment:
A full stop needs adding to the end of the sentence.

Reply:
done